keras/layers/convolutional.py

```diff
@@ -178,7 +197,9 @@ def get_config(self):
                   'b_constraint': self.b_constraint.get_config() if self.b_constraint else None,
                   'bias': self.bias,
                   'input_dim': self.input_dim,
-                  'input_length': self.input_length}
+                  'input_length': self.input_length,
+                  'W_learning_rate_multiplier': self.W_learning_rate_multiplier if self.W_learning_rate_multiplier else None,
```

Comment:
Hi Avanti,

I wanted to follow the same structure as constraints and that's way I added the ifelse. However you might be right and it seems not needed.